dz2.1/ДЗ. TypeScript. Анотація типів та примітивні типи.ts

```diff
@@ -0,0 +1,116 @@
+class School {
+  directions: Direction[];
+
+  addDirection(direction: Direction) {
```

Comment:
Please, do not forget to explicitly add type annotation for the return function value.